spec/fixtures/rails_app/Gemfile

```diff
@@ -34,7 +34,6 @@ end
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
-# For now...
-gem 'warden-jwt_auth', github: 'waiting-for-dev/warden-jwt_auth'
+gem 'warden-jwt_auth'
```

Comment:
This dependency can be safely removed from the fixture app Gemfile (my mistake). As it depends on `devise-jwt`, it already depends on the required `warden-jwt_auth` dependency.